### ISSUE TITLE
[eFEAT] reshape team analyser

### ISSUE THREAD

**Issue by akasolace** [akasolace/HO#215]
Remove player and (innacurate) stars information. In the main tab, only display ratings comparison (similar to what is done in HT). On the left side as we would have more space add additional filters.

![image](https://user-images.githubusercontent.com/1136496/64131051-68624480-cdc6-11e9-9b1d-7f133664a62b.png)

**Comment by DavidatorusF** [comment]
see the players, display how many times they have been positioned at this or that slot, display their form, display if they are injured
why not put our last known stars on our players?
capture with all possible options enabled (except mix lineup)

![](https://files.slack.com/files-pri/TH4B7A79B-FMYTJ0GA0/image.png)

**Comment by akasolace** [comment]
@yaute74  @mistermax80 @brokenelevator  what do you think about this one ?
Should we change it or keep it as it is ?

**Comment by yaute74** [comment]
We have the space to display this infos ?

**Comment by akasolace** [comment]
@yaute74  It will use less space. 
The idea was to remove all the boxes with player name and inaccurate stars information and put instead the sectors ratings, hence we can remove them from the left columns (see first screenshot). More or less a similar interface that what exist in HT but I think it is currently only a supporter feature.

**Comment by yaute74** [comment]
@akasolace I thought your question was about david  proposal 
With your proposition we will loose info as mentioned by david, no ?

**Comment by akasolace** [comment]
@yaute74, yes with my proposition we loose that info in that screen.  We alreday have such information in the player analysis tab. IMO, team analyser should be simply about providing information against opponement and to compare sector ratings. Hence, I would suggest to simplify the interface and concentrate on displaying useful rating information. 
For example, I haven't checked but I am not even sure we are comparing average rating in both case  Also I don't know what is available via CHPP, maybe some extra information are available (e.g. opponemnt average form lineup)
Again I think the team analyser is very clear in HT, it is only available to supporter but maybe we can use that as inspiration ...

**Comment by mistermax80** [comment]
@akasolace I'm not supporter, so could you show me the team analyzer in HT?
IMHO, the info in that screen are useful, I like the team analyzer in across view way, where you can see the position player opposite and the information on opposite player, in addition, I like to see other information, for ex, healt, marks, specialtis, ect, all in the following screen with across way:
![Schermata 2019-12-08 alle 01 25 23](https://user-images.githubusercontent.com/15519907/70382300-ac329000-1959-11ea-8069-26b137a6d2f4.png)

Sure:
1) the left panel with percentage, I like see it more similar to HT, I find hard read this and easy read that HT.
2) the lineup I leave asis and add additional player information where it is possibile, but I like to see better, I don't know if we can adjust color or reduce the box, I like enhance layout without lose information.
3) add information on opposite specialies and show the notes on advantage on this and the suggest how contrast it.

**Comment by akasolace** [comment]
@mistermax80 problem is that we don't have reliable info on opposite lineup. You d'ont know which players will be aligned and stars information are also unreliable. I think this why HT does not offer that functionality ...

**Comment by wsbrenk** [comment]
what about putting teamanalyzer and lineup together? not only moving teamanalyzer gui to lineup, also putting the different model classes MatchDetails, MatchKurzInfo, Match and Matchdetail together. weather prediction from regionDetails could preset rating prediction in lineup. special events could be displayed in team analyzers gui when comparing the own current lineup with recent lineups of opponent team. even man marking could be handled in team analyzers gui. 
the lineup of the opponent team should be editable to add new player or bench. the changes of the rating is hard to predict (unknown form...), but special event effects could easily be updated.

**Comment by akasolace** [comment]
@wsbrenk I think this is a too big change. I would rather stick with 2 differents tabs/modules.
Some users (like me) are hardly using team analyser. I would not like to overcomplicate the existing lineup tab.  I think it is not too difficult at the moment to go back and forth between lineup tab and match analyser to test different composition. 
My idea here was actually the opposite, it was to simplify the interface to make it more in line with HT interface by simply displaying rating information per sector ...

**Comment by mistermax80** [comment]
@akasolace yes my idea is not merge the two tabs. For the march analyzed I exposed already my opinion
